test/e2e/cypress/e2e/hosts_overview.cy.js

```diff
@@ -170,4 +170,41 @@ context('Hosts Overview', () => {
       });
     });
   });
+
+  describe('Deregistration', () => {
+    describe('Clean-up buttons should be visible only when needed', () => {
+      const heartbeatTargets = agents();
+      const failingHost = heartbeatTargets.shift();
+      before(() => {
+        cy.task('startAgentHeartbeat', heartbeatTargets);
```

Comment:
Shouldn't we navigate to the hosts view here?
I know that we do in the main `beforeAll`, but if we don't do it here, we are already basing our tests in the order of things. For example, before this describe we have the `healthDetection` tests, and these tests would already be affected by those.
What do you think?

Reply:
Good point @arbulu89 I guess that if any of the previous tests would do a `cy.visit(...)` it could break. I'll add a 
```
cy.visit('/hosts');
cy.url().should('include', '/hosts');
```
Maybe for a future PR we could change the top `before()` for a `beforeEach()` so that this URL visit/check is performed for all tests without having to add it explicitly